### PR TITLE
Register prometheus-metrics, haproxy-ocp and disk tool on openshift cluster

### PR DESCRIPTION
**Pull request by chaitanyaenr**
Registers prometheus-metrics, disk and haproxy-ocp tool on openshift cluster.

### CODE REVIEW

File: contrib/ansible/openshift/register.sh [hunk 1]
```diff
@@ -68,10 +68,13 @@ while read -u 11 line;do
     pbench-register-tool --label=$label --name=pidstat --remote $remote
     pbench-register-tool --label=$label --name=oc --remote $remote
     pbench-register-tool --label=$label --name=pprof --remote $remote -- --osecomponent=master --interval=$ose_master_interval
+    pbench-register-tool --label=$label --name=haproxy-ocp --remote $remote -- --interval=10 --counters-clear-all
```
Comment (jeremyeder) [inline contrib/ansible/openshift/register.sh]: Why is the interval hardcoded?

Comment (chaitanyaenr) [inline contrib/ansible/openshift/register.sh]: Fixed.

**Comment by chaitanyaenr** [comment]
@jeremyeder @ndokos PTAL.

File: contrib/ansible/openshift/pbench_register.yml [hunk 2]
```diff
@@ -43,7 +43,7 @@
       with_indexed_items: "{{ groups['lb'] }}"
       when: groups['lb']|default([]) 
     - name: register tools
-      shell: sh /run/pbench/register.sh {{ default_tools_interval }} {{ ose_master_interval }} {{ ose_node_interval }} {{ file }}
+      shell: sh /run/pbench/register.sh {{ default_tools_interval }} {{ ose_master_interval }} {{ ose_node_interval }} {{ file }} {{ inventory_file }}
```
Comment (jeremyeder) [inline contrib/ansible/openshift/pbench_register.yml]: does this belong here?  what does this do?

File: contrib/ansible/openshift/register.sh [hunk 3]
```diff
@@ -10,6 +10,7 @@ ose_node_interval=$3
 file=$4
 label_prefix=svt_
 hosts=/run/pbench/register.tmp.hosts
+inventory_file_path=$5
```
Comment (jeremyeder) [inline contrib/ansible/openshift/register.sh]: hmm why is this being added?  oh, prometheus tool uses the inventory. got it.

**Comment by jeremyeder** [comment]
LGTM

File: contrib/ansible/openshift/register.sh [hunk 4]
```diff
@@ -68,10 +69,14 @@ while read -u 11 line;do
...
     pbench-register-tool --label=$label --name=pprof --remote $remote -- --osecomponent=master --interval=$ose_master_interval
+    pbench-register-tool --label=$label --name=haproxy-ocp --remote $remote -- --interval=$ose_master_interval --counters-clear-all
+    pbench-register-tool --label=$label --name=disk --remote $remote
   else
     pbench-register-tool --label=$label --name=sar --remote $remote
     pbench-register-tool --label=$label --name=iostat --remote $remote
     pbench-register-tool --label=$label --name=pidstat --remote $remote
+    pbench-register-tool --label=$label --name=disk --remote $remote
+    pbench-register-tool --label=$label --name=prometheus-metrics --remote $remote -- --inventory=$inventory_file_path
   fi
```
Comment (ndokos) [inline contrib/ansible/openshift/register.sh]: You are doing a lot of the same things in the if part, as well as the else part. You should refactor the common cases out of the if/else and just deal with the differences in the if/else.

Comment (chaitanyaenr) [inline contrib/ansible/openshift/register.sh]: @ndokos My bad, didn't notice that. Fixed.

**Pull request merged**